### PR TITLE
fix(core): stop stremthru treating subtitle flags as audio languages

### PR DESCRIPTION
**Pull request by webstreamr**
getLanguages() scans the whole description for any flag emoji, so when there's no audio-line marker (🎙️), it picks up flags from the 💬 subtitle line too. Real example — Torz's first 4K YTS result for tt43657327:

```json
{
  "infoHash": "ace934e989a48837956307b1701b7ee856425def",
  "name": "[P2P]\nTorz\n4k",
  "description": "💿 WEB 🎞️ HEVC\n🎧 AAC | 5.1\n📦 4.4 GB 〽️ 800 KB/s\n💬 🇬🇧 🇸🇦 🇵🇹 🇭🇷 🇨🇿 🇩🇰 🇳🇱 🇪🇸 🇫🇮 🇫🇷 🇩🇪 🇬🇷 🇮🇱 🇭🇺 🇮🇩 🇮🇹 🇯🇵 🇰🇷 🇲🇾 🇵🇱 🇷🇴 🇷🇺 🇨🇳 🇸🇪 🇹🇭 🇹🇷 🇺🇦 🇻🇳\n📄 Big.Chicken.A.Fast.Food.Conspiracy.2026.2160p.4K.WEB.x265.10bit.AAC5.1-[YTS.GG - YTS.BZ].mkv",
  ...
}
```

No 🎙️/🌐 line — StremThru's probe found zero audio tracks, so it correctly omits the line. AIOStreams then fell back to scanning the whole description and misread the 28 subtitle flags as spoken languages.

getLanguages() now returns `[]` unless 🌐 is present. 🌐 is ambiguous (StremThru also uses it for an unprobed filename guess), so only then does it fall back to the generic scan. 🎙️/💬 extraction is unchanged.

Flag extraction is consolidated into a single getLanguagesAfterMarker() helper (fixes indicator escaping and non-space whitespace between flags along the way), with tests. Verified against 1,585 real Torz + 227 real Meteor results across 5 titles.

## Summary by CodeRabbit

* **New Features**
  * Improved language detection from flag emojis in stream descriptions.
  * Added support for identifying multiple audio and subtitle languages following their respective markers.
  * Standardised language names and handled unsupported flags gracefully.

* **Bug Fixes**
  * Improved handling of missing markers, whitespace, null content and special characters during language detection.

* **Tests**
  * Added coverage for recognised, unknown and unmapped flags, multiple languages and edge cases.

### CODE REVIEW

File: packages/core/src/parser/utils.ts [hunk 1]
```diff
@@ -324,3 +324,11 @@ export function extractInfoHashFromMagnet(magnet: string): string | undefined {
   if (match.length === 40) return match.toLowerCase();
   return base32ToHex(match);
 }
+
+/** Matches one or more flag emojis (each two regional indicator chars) after an indicator emoji. */
+export function getFlagRegex(indicator: string): RegExp {
+  return new RegExp(
+    `${indicator}\\s*((?:[\\u{1F1E6}-\\u{1F1FF}]{2}\\s*)+)`,
+    'u'
```
[the remark this answers is not in the document]
Comment (webstreamr) [reply packages/core/src/parser/utils.ts]: done

File: packages/core/src/presets/meteor.ts [hunk 2]
```diff
@@ -10,12 +10,23 @@ import {
...
     return ['🔗', '📰'];
   }
+
+  // Scoped to 🌐 so 💬 (subtitle) flags never leak in as audio languages.
+  protected override getLanguages(stream: Stream): string[] {
+    const audioMatch = stream.description?.match(getFlagRegex('🌐'));
+    if (!audioMatch) return [];
+    return audioMatch[1]
+      .split(' ')
+      .map((part) => this.convertFlagToLanguage(part.trim()))
```
[the remark this answers is not in the document]
Comment (webstreamr) [reply packages/core/src/presets/meteor.ts]: done

File: packages/core/src/presets/stremthru.ts [hunk 3]
```diff
@@ -46,37 +46,38 @@ export class StremThruStreamParser extends StreamParser {
...
   }
 
+  // 🎙️/💬 are handled in getParsedFileMergeOverrides. 🌐 is ambiguous (probe
+  // vs. filename guess), so fall back to the generic scan for it.
+  protected override getLanguages(
+    stream: Stream,
+    currentParsedStream: ParsedStream
+  ): string[] {
+    if (!stream.description?.includes('🌐')) return [];
+    return super.getLanguages(stream, currentParsedStream);
```
[the remark this answers is not in the document]
Comment (webstreamr) [reply packages/core/src/presets/stremthru.ts:56]: configurable stream templates are not supported in the torz built-in but they are also not supported in torz itself atm.
